src/shared/components/error-pages/__tests__/NoResultsPage.spec.tsx

```diff
@@ -4,6 +4,11 @@ import customRender from '../../../__test-helpers__/customRender';
 
 import NoResultsPage from '../full-pages/NoResultsPage';
 
+jest.mock('../../error-component/ErrorBoundary', () => ({
```

Comment:
Would it be worth mocking this globally just once as it's used in several places?

Reply:
I thought about that too but the problem on this one is that there are places where we don't want to mock this because the test is depending on something within the error boundary, and I think it changed too many snapshots that I wasn't willing to check at this time.
So yeah, at the moment I guess it's better to go for an ad-hoc approach